parts/k8s/kubernetesmastercustomdatavmss.yml

```diff
@@ -0,0 +1,540 @@
+#cloud-config
```

Comment:
What's so different about the master vmss cloud-init implementation that justifies a distinct file to maintain?

Reply:
compared to the existing (vmas) one, that is

Reply:
+1

Reply:
There are lots of things that are quite different. Mostly because we use offset everywhere for VMAS and VMSS is all derived. There is alot of variables used in one but not the other and vice versa. I was debating with myself if it's better to keep them in the same file with lots of nested if/elses or if it's better to keep them separated, which is cleaner and easier to test things. Can you pls take a look at the diff for  `kubernetesmastercustomdatavmss.yml` and `kubernetesmastervarsvmss.t` and let me know what you think is the best.

Reply:
I'm O.K. w/ separate files for now. There's probably a better way to decompose this overall as "view layer" components, but don't want that burden on this PR.